releasenotes/notes/add-grover-primitives-10f81efdba93703d.yaml

```diff
@@ -0,0 +1,26 @@
+---
+features:
+  - |
+    :class:`~.Grover` supports the primitives and can use :class:`~.BaseSampler` to
+    calculate the results. Accordingly, ``quantum instance`` in :class:`~.Grover`
+    is pending deprecation and will be deprecated in a future release.
+
+    Example::
+    .. code-block:: python
```

Comment:
```suggestion
    Example:

    .. code-block:: python
```
This needed fixing - sorry if that was unclear in the other comment. Otherwise it wraps it all in a single paragraph, thus

![image](https://user-images.githubusercontent.com/40241007/191652298-c59cd752-6cc4-4df5-95ba-44eb45168e4e.png)